### PR TITLE
Tests created

### PR DESCRIPTION
**Pull request by Gredja**

### CODE REVIEW

File: Softeq.NetKit.Chat.Tests/Unit/Domain.Services/ChannelService/MuteChannelAsyncTests.cs [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+            // Assert
+            act.Should().Throw<NetKitChatNotFoundException>()
+                .And.Message.Should().Be($"Unable to mute channel. Member {nameof(saasUserId)}:{saasUserId} not found.");
+
+            VerifyMocks();
+        }
+
+        [Fact]
+        public void ShouldThrowIfMemberDoesNotInChannel()
```
Comment (eugenekirvel) [inline Softeq.NetKit.Chat.Tests/Unit/Domain.Services/ChannelService/MuteChannelAsyncTests.cs]: ShouldThrowIfMemberIsNotInChannel

File: Softeq.NetKit.Chat.Tests/Unit/Domain.Services/ChannelService/MuteChannelAsyncTests.cs [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+            // Assert
+            act.Should().Throw<NetKitChatInvalidOperationException>()
+                .And.Message.Should().Be($"Unable to mute channel. Member saasUserId:{member.SaasUserId} is not joined channel {nameof(channelId)}:{channelId}.");
+
+            VerifyMocks();
+        }
+
+        [Fact]
+        public async Task ShouldReturnTask()
```
Comment (eugenekirvel) [inline Softeq.NetKit.Chat.Tests/Unit/Domain.Services/ChannelService/MuteChannelAsyncTests.cs]: check that  channel is muted

File: Softeq.NetKit.Chat.Tests/Unit/Domain.Services/ChannelService/MuteChannelAsyncTests.cs (hunk 1 above)
Comment (Gredja) [reply Softeq.NetKit.Chat.Tests/Unit/Domain.Services/ChannelService/MuteChannelAsyncTests.cs]: Renamed

File: Softeq.NetKit.Chat.Tests/Unit/Domain.Services/ChannelService/MuteChannelAsyncTests.cs (hunk 2 above)
Comment (Gredja) [reply Softeq.NetKit.Chat.Tests/Unit/Domain.Services/ChannelService/MuteChannelAsyncTests.cs]: Additional check

File: Softeq.NetKit.Chat.Tests/Unit/Domain.Services/ChannelService/MuteChannelAsyncTests.cs [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+            _memberRepositoryMock.Setup(x => x.GetMemberBySaasUserIdAsync(It.IsAny<string>()))
+                .ReturnsAsync((Member)null)
+                .Verifiable();
+
+            // Act
+            Func<Task> act = async () => { await _channelService.MuteChannelAsync(saasUserId, channelId, isMuted); };
+
+            // Assert
+            act.Should().Throw<NetKitChatNotFoundException>()
+                .And.Message.Should().Be($"Unable to mute channel. Member {nameof(saasUserId)}:{saasUserId} not found.");
```
Comment (eugenekirvel) [inline Softeq.NetKit.Chat.Tests/Unit/Domain.Services/ChannelService/MuteChannelAsyncTests.cs]: is not found

**Pull request merged**